mirage/config.js

```diff
@@ -197,5 +207,8 @@ export default function() {
       }
     }]
   })
-  this.passthrough('http://165.227.76.52:5000/**');
+
+  if (ENV.environment !== "test") {
```

Comment:
If the environment is test, we don't want any api calls going to our real backend